package.json

```diff
@@ -110,7 +110,7 @@
     "rollup-plugin-polyfill-node": "0.9.0",
     "rollup-plugin-postcss": "4.0.2",
     "rollup-plugin-string": "3.0.0",
-    "rollup-plugin-svgo": "1.1.0",
+    "rollup-plugin-svgo": "^2.0.0",
```

Comment:
Why were these packages updated/version strings changed? If these are not related to this PR, we should move them to a separate PR.